B/Bullet/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Bullet"
+version = v"3.22.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/bulletphysics/bullet3/archive/refs/tags/3.22b.tar.gz",
+                  "c6cd89ecbc4bd73fee64723c831c1578daab056d88774755a6f56afc6f417b2b")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/bullet*
+mkdir build && cd build
+cmake -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_CXX_STANDARD="11" \
+    -DBUILD_PYBULLET=OFF \
+    -DBUILD_UNIT_TESTS=OFF \
+    -DBUILD_CPU_DEMOS=OFF \
+    -DBUILD_OPENGL3_DEMOS=OFF \
+    -DBUILD_BULLET2_DEMOS=OFF \
+    -DBUILD_EXTRAS=OFF \
+    -S .. \
+    -B .
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
```

Comment:
We don't get the warnings during audit because these are static libraries, but we do have C++ string ABI issues:
```console
% nm lib/*.a | c++filt | grep basic_string
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_leak_hard()
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::_Rep::_M_destroy(std::allocator<char> const&)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::_Rep::_M_dispose(std::allocator<char> const&)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::_Rep::_S_empty_rep_storage
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(char const*, std::allocator<char> const&)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(std::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::_Rep::_M_destroy(std::allocator<char> const&)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::_Rep::_M_dispose(std::allocator<char> const&)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::_Rep::_S_empty_rep_storage
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(char const*, std::allocator<char> const&)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(std::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::compare(char const*) const
                 U std::basic_stringbuf<char, std::char_traits<char>, std::allocator<char> >::str() const
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_leak_hard()
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::_Rep::_M_destroy(std::allocator<char> const&)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::_Rep::_M_dispose(std::allocator<char> const&)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::_Rep::_S_empty_rep_storage
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::assign(char const*, unsigned long)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::resize(unsigned long, char)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::reserve(unsigned long)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_mutate(unsigned long, unsigned long, unsigned long)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(char const*, unsigned long, std::allocator<char> const&)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(char const*, std::allocator<char> const&)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(std::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned long, unsigned long)
                 U std::basic_stringbuf<char, std::char_traits<char>, std::allocator<char> >::_M_sync(char*, unsigned long, unsigned long)
0000000000000000 W std::basic_stringbuf<char, std::char_traits<char>, std::allocator<char> >::~basic_stringbuf()
0000000000000000 W std::basic_stringbuf<char, std::char_traits<char>, std::allocator<char> >::~basic_stringbuf()
0000000000000000 W std::basic_stringbuf<char, std::char_traits<char>, std::allocator<char> >::~basic_stringbuf()
0000000000000000 n std::basic_stringbuf<char, std::char_traits<char>, std::allocator<char> >::~basic_stringbuf()
                 U std::basic_stringstream<char, std::char_traits<char>, std::allocator<char> >::basic_stringstream(std::_Ios_Openmode)
                 U std::basic_stringstream<char, std::char_traits<char>, std::allocator<char> >::~basic_stringstream()
                 U std::basic_istream<char, std::char_traits<char> >& std::getline<char, std::char_traits<char>, std::allocator<char> >(std::basic_istream<char, std::char_traits<char> >&, std::basic_string<char, std::char_traits<char>, std::allocator<char> >&, char)
                 U VTT for std::basic_stringstream<char, std::char_traits<char>, std::allocator<char> >
                 U vtable for std::basic_stringbuf<char, std::char_traits<char>, std::allocator<char> >
                 U vtable for std::basic_stringstream<char, std::char_traits<char>, std::allocator<char> >
0000000000002ff0 T btReducedDeformableBodyHelpers::createReducedDeformableObject(btSoftBodyWorldInfo&, std::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, int, bool)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::compare(char const*) const
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::_Rep::_M_destroy(std::allocator<char> const&)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::_Rep::_M_dispose(std::allocator<char> const&)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::_Rep::_S_empty_rep_storage
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::append(char const*, unsigned long)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::append(std::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(char const*, unsigned long, std::allocator<char> const&)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(char const*, std::allocator<char> const&)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(std::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
                 U std::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(std::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned long, unsigned long)
                 U std::basic_stringbuf<char, std::char_traits<char>, std::allocator<char> >::_M_sync(char*, unsigned long, unsigned long)
0000000000000000 W std::basic_stringbuf<char, std::char_traits<char>, std::allocator<char> >::~basic_stringbuf()
0000000000000000 W std::basic_stringbuf<char, std::char_traits<char>, std::allocator<char> >::~basic_stringbuf()
0000000000000000 W std::basic_stringbuf<char, std::char_traits<char>, std::allocator<char> >::~basic_stringbuf()
0000000000000000 n std::basic_stringbuf<char, std::char_traits<char>, std::allocator<char> >::~basic_stringbuf()
                 U std::basic_stringstream<char, std::char_traits<char>, std::allocator<char> >::~basic_stringstream()
                 U std::basic_istream<char, std::char_traits<char> >& std::getline<char, std::char_traits<char>, std::allocator<char> >(std::basic_istream<char, std::char_traits<char> >&, std::basic_string<char, std::char_traits<char>, std::allocator<char> >&, char)
                 U VTT for std::basic_stringstream<char, std::char_traits<char>, std::allocator<char> >
                 U vtable for std::basic_stringbuf<char, std::char_traits<char>, std::allocator<char> >
                 U vtable for std::basic_stringstream<char, std::char_traits<char>, std::allocator<char> >
```
```suggestion
platforms = expand_cxxstring_abis(supported_platforms())
```